component/Header.tsx

```diff
@@ -1,15 +1,15 @@
-import getConfig from 'next/config'
 import Image from 'next/image'
-import { useRouter } from 'next/router'
 import React from 'react'
+import getConfig from 'next/config'
+import { useRouter } from 'next/router'
 
 const { publicRuntimeConfig } = getConfig()
 
-const Header = () => {
+const Header = ({fixedPosition = true}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fixedPosition` prop is set to `true` by default in the new hunk, which means the header will be fixed by default. This is a change in behavior from the old hunk where the header was always fixed. If this change is intentional and all uses of the `Header` component have been checked to ensure they work correctly with this new default, then it's fine. Otherwise, it might be safer to default `fixedPosition` to `false` to maintain the previous behavior when the prop is not provided.
```diff
- const Header = ({fixedPosition = true}) => {
+ const Header = ({fixedPosition = false}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->